releasenotes/notes/sentence-window-retriever-output-docs-d3de2ac4328488f1.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    The output of the `SentenceWindowRetriever` now includes the document text and the document metadata.
+    This makes it easier to understand the context of the retrieved sentences.
```

Comment:
I think the release note should be clearer and similar to the PR description.

Reply:
made it shorter and verbatim from the PR title

Reply:
I think that something similar to this would be clearer:
```python
- `context_windows`: 
    A list of strings, where each string represents the concatenated text from the context window of the 
    corresponding document in `retrieved_documents`.
- `context_documents`: 
    A list of lists of `Document` objects, where each inner list contains the documents that form the 
    context window for the corresponding document in `retrieved_documents`.
```